openstack/loadbalancer/v2/l7policies/results.go

```diff
@@ -0,0 +1,93 @@
+package l7policies
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// L7Policy is a collection of L7 rules associated with a Listener, and which
+// may also have an association to a back-end pool.
+type L7Policy struct {
+	// The unique ID for the L7 policy.
+	ID string `json:"id"`
+
+	// Name of the L7 policy.
+	Name string `json:"name"`
+
+	// The ID of the listener.
+	ListenerID string `json:"listener_id"`
+
+	// The L7 policy action. One of REDIRECT_TO_POOL, REDIRECT_TO_URL, or REJECT.
+	Action Action `json:"action"`
```

Comment:
Let's change this to `string`. Per the comment in the prior PR, custom types are only for requests. Results only use basic types (except if it's a struct).